napari/_qt/menus/file_menu.py

```diff
@@ -173,3 +179,10 @@ def _add_sample(*args, plg=plugin_name, smp=samp_name):
 
                 menu.addAction(action)
                 action.triggered.connect(_add_sample)
+
+    def update(self):
```

Comment:
for consistency, do you want to make that `def update(self, event=None)`?

Reply:
Fixed!

Reply:
I think you forgot to push.